Backpack/src/main/java/net/skyscanner/backpack/toggle/BpkSwitch.kt

```diff
@@ -58,11 +56,16 @@ open class BpkSwitch @JvmOverloads constructor(
     val textDisabledColor = context.getColor(R.color.bpkSkyGrayTint04)
     val textEnabledColor = context.getColor(R.color.bpkTextPrimary)
     context.theme.obtainStyledAttributes(attrs, R.styleable.BpkSwitch, defStyleAttr, 0).use {
-      val checkedColor = it.getColor(R.styleable.BpkSwitch_switchPrimaryColor, context.getColor(R.color.bpkPrimary))
-      val trackCheckedColor = ColorUtils.setAlphaComponent(checkedColor, CHECKED_TRACK_COLOR_ALPHA)
+      val primaryColor = context.getColor(R.color.bpkPrimary)
+      val checkedColor = it.getColor(R.styleable.BpkSwitch_switchPrimaryColor, primaryColor)
+      val trackCheckedColor = if (checkedColor == primaryColor) {
```

Comment:
I thought about introducing a theme colour for this as well, but it'd be a breaking change and something that might be easily missed, so decided to keep using the calculated version here unless it's the default theme